### ISSUE TITLE
Why not change riot.quiet into riot.make_defs

### ISSUE THREAD

**Issue by keestux** [attdona/riot-code#10]
Right now we have `riot.quiet` which is supposed to be filled in with `QUIET=0` or `QUIET=1`.

May I suggest to change `riot.quiet` and name it `riot.make_defs`, or `riot.build_defs` or something else. The default can still be `QUIET=0` or `QUIET=1`.

This setting can then be used to pass one or more defines on the make command line. For example,
```
		"riot.make_defs": "QUIET=1 USB_VID=2341 USB_PID=804d",
```

**Comment by attdona** [comment]
Changed to `riot.make_defs` and just published riot-code v0.1.4.

Greetings
Attilio

**Comment by keestux** [comment]
Sorry, your change didn't bring what I was hoping for. The problem is that not all make commands are created the same.

The following is fine (default build)
```
    let make_cmd =
    project.compiler_path === ''
      ? 'make ${config:riot.make_defs} BOARD=${config:riot.board}'
      : 'PATH=${config:riot.compiler_path}:$PATH make ${config:riot.make_defs} BOARD=${config:riot.board}'
```

But these aren't (i.e. "clean" and "flash")
```
        args: ['${config:riot.make_defs}', 'BOARD=${config:riot.board}', 'flash'],
```
and
```
        args: ['${config:riot.make_defs}', 'BOARD=${config:riot.board}', 'clean'],
```
In the latter two the `make_defs` ends up as one argument in the make command, surrounded by single quotes, I think.

**Comment by attdona** [comment]
Thanks for reporting!

Fixed in the 0.1.5 version.

After updating riot-code `RIOT init` command is necessary to rebuild `tasks.json`.
